index.js

```diff
@@ -36,12 +50,18 @@ window.onload = () => {
 
     if(location.host === urls.prod) {
         !firebase.apps.length ? firebase.initializeApp(prodFirebaseConfig()) : firebase.app();
+        window.DD_RUM && window.DD_RUM.init({ ...DatadogDevConfig, env: 'prod' });
+        window.DD_RUM && window.DD_RUM.startSessionReplayRecording();
     }
     else if(location.host === urls.stage) {
         !firebase.apps.length ? firebase.initializeApp(stageFirebaseConfig()) : firebase.app();
+        window.DD_RUM && window.DD_RUM.init({ ...DatadogDevConfig, env: 'stage' });
+        window.DD_RUM && window.DD_RUM.startSessionReplayRecording();
     }
     else {
         !firebase.apps.length ? firebase.initializeApp(devFirebaseConfig()) : firebase.app();
+        window.DD_RUM && window.DD_RUM.init(DatadogDevConfig);
+        window.DD_RUM && window.DD_RUM.startSessionReplayRecording();
```

Comment:
Could we pull this line outside of this IF-ELSE block and just declare it once?

Reply:
Updated.